events/metrics.go

```diff
@@ -0,0 +1,36 @@
+package events
+
+import (
+	pmetrics "github.com/awslabs/operatorpkg/metrics"
+	"github.com/prometheus/client_golang/prometheus"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+const (
+	MetricLabelGroup       = "group"
+	MetricLabelKind        = "kind"
+	MetricLabelEventType   = "type"
+	MetricLabelEventReason = "reason"
+)
+
+const (
+	MetricNamespace = "operator"
+	MetricSubsystem = "event"
```

Comment:
Should this subsystem be prefixed with the object name? We already are thinking about doing this same thing for our status condition controller so should we do the same thing here?